src/store/modules/product/actions.ts

```diff
@@ -44,9 +44,15 @@ const actions: ActionTree<ProductState, RootState> = {
   async getProductInformation ({ dispatch }, { orders }) {
     let productIds: any = new Set();
     orders.forEach((order: any) => {
-      order.items.forEach((item: any) => {
-        if (item.itemId) productIds.add(item.itemId);
-      });
+      if(order.items) {
+        order.items.forEach((item: any) => {
+          if (item.itemId) productIds.add(item.itemId);
+        });
+      } else {
+        order.doclist.docs.forEach((item: any) => {
```

Comment:
I think we should move this logic to calling action along with dispatch
dispatch('fetchProducts', { productIds })
dispatch('stock/addProducts', { produc tIds })

Reply:
Changed logic to destructure productIds from orders and call actions from order module itself.